app/assets/javascripts/utilities/slideSidebar.js

```diff
@@ -2,23 +2,43 @@ function slideSidebar(side, direction) {
   if (!document.getElementById('sidebar-wrapper-' + side)) {
     return;
   }
+
+  const articlesList = document.getElementById('articles-list');
+  const mainContent = document.getElementById('main-content');
+
   if (direction === 'intoView') {
-    document.getElementById('articles-list').classList.add('modal-open');
+    if (articlesList) {
```

Comment:
refactor (potentially blocking): Are these changes and other event listener management changes required for the skip link refactor? If not, create a separate PR for these changes.

Reply:
Yeah I think I've connected the dots too early on this one - we will need to make changes here eventually (since the 'articles-list' will become 'main-content'), but it seems the `slideSidebar` code is actually only being used on the tag index page, so this should wait until that page is given the skip link feature 👍